qemu/tests/cfg/x86_cpu_flags.cfg

```diff
@@ -124,3 +121,17 @@
                     test_dir = '/home'
                     source_file = 'cpuid.src.tar.gz'
                     check_cpuid = './cpuid -r | grep eax=0x40000001 | uniq -c'
+                - memory_protection_key:
+                    flags = "pku ospke"
+                    variants:
+                        - @default_model:
+                            cpu_model_flags += ",+pku"
+                            HostCpuVendor.amd:
+                                no RHEL.6
+                                no Host_RHEL.m8.u2
```

Comment:
Not only 8.2 host, but also all hosts before 8.2, right?

Reply:
update, thanks.